README.md

```diff
@@ -147,8 +147,40 @@ sudo dpkg -i postgresql-9.3-tsearch-extras_0.1.2_amd64.deb
 sudo apt-get install postgresql-9.4
 wget https://dl.dropboxusercontent.com/u/283158365/zuliposs/postgresql-9.4-tsearch-extras_0.1_amd64.deb
 sudo dpkg -i postgresql-9.4-tsearch-extras_0.1_amd64.deb
+```
+
+On Fedora 22:
 
-# Then, all versions:
+```
+sudo dnf install libffi-devel memcached rabbitmq-server openldap-devel python-devel redis postgresql-server postgresql-devel postgresql libmemcached-devel
+wget https://launchpad.net/~tabbott/+archive/ubuntu/zulip/+files/tsearch-extras_0.1.3.tar.gz
+tar xvzf tsearch-extras_0.1.3.tar.gz
+cd ts2
+make
+sudo make install
+
+# now create the missing dictionary files
+sudo touch /usr/share/pgsql/tsearch_data/english.stop
+sudo touch /usr/share/pgsql/tsearch_data/en_us.dict
+sudo touch /usr/share/pgsql/tsearch_data/en_us.affix
```

Comment:
We should probably figure out how to replace this with the correct actual dictionary files.  

On Ubuntu we get the dictionaries by installing the hunspell-en-us package, and then there's a program pg_updatedicts that converts that into postgres dict files under /var/cache/postgresql/dicts/, which our configuration then symlinks into /usr/share/postgresql/9.x/tsearch_data/

Note that for the stopwords file in particular, we have a custom version in the zulip repo at puppet/zulip/files/postgresql/zulip_english.stop that should be installed in that directory.  There's code to do this in provision.py, for example.